### DISCUSSION TITLE
XSPICE Cores Library revisited

### DISCUSSION THREAD

**Discussion by tomhajjar** [ra3xdh/qucs_s#656] [Q&A]
1. Cores.lib is not "visually" Blacklisted in Qucsator mode. The lib itself is however.
2. Originally I thought the cores data was all alike but only the H array is. The B array is different for each core. The data in Mike Brinson's Test_Cores_prj is different for every core test schematic. 

![2024-03-22_083502](https://github.com/ra3xdh/qucs_s/assets/102838604/90d24b32-0395-4340-a328-496c220cb15c)
![2024-03-22_083445](https://github.com/ra3xdh/qucs_s/assets/102838604/4f8ac7c1-e728-42c5-a62f-01c4710c7d77)

**Comment by ra3xdh** [comment]
> Cores.lib is not "visually" Blacklisted in Qucsator mode. The lib itself is however.

The simulator compatibility of library devices is not checked in schematic. It is not possible to implement without library format extension and revising of every device in every library. 

> Originally I thought the cores data was all alike but only the H array is.

Yes, the B-array seems to be correct.

**Comment by ra3xdh** [comment]
Moving this issue to discussion: https://github.com/ra3xdh/qucs_s/discussions
